Tryouts/Plugins/ViewViewModelPlugins/Authentication/AuthenticationView/Views/AuthenticationView.axaml

```diff
@@ -0,0 +1,66 @@
+<!--
```

Comment:
Ok it sounds like my assumptions around the value this project was meant to provide are not inline with your intentions.  Having readme.md files for project stating what they are meant to illustrate and what their value is would help with not having misunderstandings like this.

Regarding your response, I think you are implying things here without stating it explicitly.  Let me tell you what I assume you mean by what you say.
"Compose will have a (or multiple) Shell implementation(s) that will support WPF, Avalonia and HTML (not JavaScript) views in the same Shell.  Compose will provide guidelines that WPF and Avalonia Views should be DataTemplate based (what about HTML?), and that WPF and Avalonia Views should not be mixed in the same project (why?)."

I have a couple questions inline there with my assumptions.  Additional to that I will state that if my assumptions are correct, then we should be having conversations around them.  Maybe this project was meant to partially drive these discussions, but that wasn't clear.

Reply:
The clients will be able to add any views - whether as DataTemplates or as UserControls both in WPF and Avalonia. Avalonia, WPF and Html views can mix (at least Avalonia can contain a WPF view and both can contain Html) - I guess I did not state it correctly - it is not 'either or', but in fact there can be mix of technologies within the same view. 
DataTemplates are more advisable to build the views since it discourages a code-behind anti-pattern. So building a view as a DataTemplate is our advice to the client, not written in stone - the clients are free to build it the way they want. I am going to add this information as a Readme file to our Prototype View project (the only one remaining after the cleanup).